ospfd/ospf_abr.c

```diff
@@ -434,6 +434,13 @@ void ospf_abr_nssa_check_status(struct ospf *ospf)
 						zlog_debug(
 							"ospf_abr_nssa_check_status: elected translator");
 				} else {
```

Comment:
I guess I'm a little confused by the flow here ... to make certain I understand ... if we find we are not elected, but we were previously the translator, we set ourselves to translating with a timer so we stop once the timer wakes up, correct?

Reply:
@riw777 correct, this is my understanding from RFC 3101: _If an elected translator determines its services are no longer required, it continues to perform its translation duties for the additional time interval defined by a new area configuration parameter, TranslatorStabilityInterval_

Reply:
thanks!